PrivateHeaders/APPX/ZIP.h

```diff
@@ -182,6 +184,44 @@ namespace appx {
         }
     };
 
+    inline
+    bool
+    _IsAPPXFile(const std::string &inputFileName)
```

Comment:
It's a private method, where is the documentation that shows how to name private methods?